src/actions/commentVote/index.ts

```diff
@@ -13,12 +13,18 @@ const voteHandler = async (
   data: InputTypeHandleVote,
 ): Promise<ReturnTypeHandleVote> => {
   const session = await getServerSession(authOptions);
```

Comment:
ideally should take weather it is a vote or downvote as an input
Toggling can lead to inconsistencies if the user is in two tabs/if the tab hasn't updated

Reply:
good follow up to add @siinghd